runtime/src/bank.rs

```diff
@@ -6902,6 +6902,9 @@ impl TransactionProcessingCallback for Bank {
             .unwrap_or(0)
     }
 
+    // Overrides default TransactionProcessingCallback::calculate_fee() to calculate actual transaction fee;
+    // Checking for `zero_fees_for_test` is done at callsite (eg. transaction_processor) where blockhash_queue's
+    // lamports_per_siganture is checked ` == 0`.
```

Comment:
do we want to add comments for something we want to remove?

Reply:
rn, I am trying to remove `fee_rate_governor` from `bank`; removing `zero_fees_for_test` is a different track -- mainly involving tests updates. Before we get to that, some notes to keep sanity 😃